packages/common/src/services/audius-backend/solana.ts

```diff
@@ -351,11 +351,12 @@ export const pollForBalanceChange = async (
 }
 
 export type PurchaseContentArgs = {
-  id: number
+  id: ID
   blocknumber: number
   extraAmount?: number | BN
   type: 'track'
   splits: Record<string, number | BN>
+  purchaserUserId?: ID
```

Comment:
If we update all flows to pass this (which it sounded like we were going to do), it should be required.

Reply:
I agree, just made this change